src/app/pages/questions/components/question/info-screen/info-screen.component.ts

```diff
@@ -47,15 +47,25 @@ export class InfoScreenComponent implements OnInit {
       })
     })
 
-    // NOTE: Save timestamp (epoch) and activate the next button
-    const epoch: number = new Date().getTime()
-    this.valueChange.emit(epoch)
-    this.showScrollButton = this.sections.length > 1
+    if (this.sections.length > 1) this.showScrollButton = true
+    else this.emitTimestamp()
   }
 
   scrollDown() {
     const dimensions = this.content.getContentDimensions()
-    const position = dimensions.scrollTop + dimensions.contentHeight / 2
+    const position = dimensions.scrollTop + dimensions.contentHeight
     this.content.scrollTo(0, position, 1000)
   }
+
+  onScroll(event) {
+    if (event.scrollTop >= event.scrollHeight - event.contentHeight) {
```

Comment:
why do we handle this here like this instead of defining the value for scrollY?

Reply:
Ionic has its own scroll events that gives how much the user has scrolled `(event.ScrollTop)` and depending on the screen height, the `scrollHeight` and `contentHeight` would vary.